build/pkgs/sphinx_inline_tabs/type

```diff
@@ -0,0 +1 @@
+standard
```

Comment:
Adding a new standard package needs a period where it is optional and then a vote on the mailing list first, right?

Reply:
We don't do that for trivialities.